cmd/auth.go

```diff
@@ -34,6 +35,11 @@ func AuthCmd(fsets ...*flag.FlagSet) *cobra.Command {
 				return err
 			}
 
+			ttl, err := cmd.Flags().GetDuration("ttl")
```

Comment:
Should we have a big default here or require a well-defined value all the time?

Reply:
Id short-circuit immediately if TTL is not passed. I don't see any reason why we should pick TTL for the user.

Reply:
+ to slava's comment. We should not set a TTL - our default should be that the key is eternal unless TTL is specified.

Reply:
Force them to pass, so they know when to expect expiry